### ISSUE TITLE
can't convert Array into String

### ISSUE THREAD

**Issue by iamlacroix** [HipByte/motion-testflight#1]
I was receiving the following error when attempting to build while using the motion-testflight gem:

```
rake aborted!
can't convert Array into String
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/gems/1.9.1/gems/motion-testflight-1.1/lib/motion/project/testflight.rb:73:in `expand_path'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/gems/1.9.1/gems/motion-testflight-1.1/lib/motion/project/testflight.rb:73:in `block in create_launcher'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/gems/1.9.1/gems/motion-testflight-1.1/lib/motion/project/testflight.rb:73:in `each'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/gems/1.9.1/gems/motion-testflight-1.1/lib/motion/project/testflight.rb:73:in `find'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/gems/1.9.1/gems/motion-testflight-1.1/lib/motion/project/testflight.rb:73:in `create_launcher'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/gems/1.9.1/gems/motion-testflight-1.1/lib/motion/project/testflight.rb:48:in `team_token='
/Users/michael/Projects/iOS/FiveLanguagesAppreciation/Appreciation/Rakefile:16:in `block in <top (required)>'
/Library/RubyMotion/lib/motion/project/app.rb:45:in `block in setup'
/Library/RubyMotion/lib/motion/project/app.rb:45:in `each_value'
/Library/RubyMotion/lib/motion/project/app.rb:45:in `setup'
/Users/michael/Projects/iOS/FiveLanguagesAppreciation/Appreciation/Rakefile:7:in `<top (required)>'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/1.9.1/rake/rake_module.rb:25:in `load'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/1.9.1/rake/rake_module.rb:25:in `load_rakefile'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/1.9.1/rake/application.rb:501:in `raw_load_rakefile'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/1.9.1/rake/application.rb:82:in `block in load_rakefile'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/1.9.1/rake/application.rb:133:in `standard_exception_handling'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/1.9.1/rake/application.rb:81:in `load_rakefile'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/1.9.1/rake/application.rb:65:in `block in run'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/1.9.1/rake/application.rb:133:in `standard_exception_handling'
/Users/michael/.rbenv/versions/1.9.3-p125/lib/ruby/1.9.1/rake/application.rb:63:in `run'
/Users/michael/.rbenv/versions/1.9.3-p125/bin/rake:32:in `<main>'
```

I was able to resolve this by setting the testflight options before I included any 3rd party libraries. Not much of an issue, but wanted to post this in the event someone else runs into it. Here's an example of how I set up my app config block:

``` ruby
Motion::Project::App.setup do |app|
  app.name = 'App Name'

  # TestFlight
  app.testflight.sdk = 'vendor/TestFlightSDK'
  app.testflight.api_token = ENV['TEST_FLIGHT_API_TOKEN']
  app.testflight.team_token = ENV['TEST_FLIGHT_TEAM_TOKEN']

  # Include 3rd party libs
  app.files.unshift(Dir.glob(File.join(app.project_dir, 'vendor/BubbleWrap/lib/**/*.rb')))

  # ...
end
```

**Comment by siuying** [comment]
+1 on this issue

**Comment by rapind** [comment]
I ran into this issue today. Thanks for providing the workaround. At a minimum, the documentation should be updated.

**Comment by siuying** [comment]
Should have been fixed in #9 (ffb07f2ad), can you confirm?

**Comment by Watson1978** [comment]
Indeed, looks this is same issue with #9.
So, I will close this issue. If you reproduce this issue, feel free to reopen this :)

Thank you.
